README.md

```diff
@@ -11,9 +11,15 @@
 var fs = require('fs');
 var pdf = require('./lib');
 var html = fs.readFileSync('./test/businesscard.html', 'utf8')
-pdf.create(html, { width: '50mm', height: '90mm'}, function(err, buffer) {
+pdf.create(html, { filename: './businesscard.pdf', width: '50mm', height: '90mm'}, function(err, res) {
   if (err) return console.log(err);
-  fs.writeFile('businesscard.pdf', buffer);
+  console.log(res);
+  /*
+    {
+      filename: './businesscard.pdf',
```

Comment:
Added in https://github.com/Atinux/node-html-pdf/commit/e24ebfa3aa5aaa7b94920e4ff2d1de351ab3e1c8